backend/config/curation-api.yml

```diff
@@ -1,12 +1,19 @@
 openapi: 3.0.0
 info:
   version: "1.0.0"
-  title: Chan Zuckerberg Initiative cellxgene Data Portal Curator API
+  title: Chan Zuckerberg Initiative CELLxGENE Discover Curator API
```

Comment:
I think we should strike Curator at this point because as noted, the API is for both curators AND downloaders. Long term it's the replacement for the V0 API that offered a subset of this functionality.